integration/operator_test.go

```diff
@@ -13,10 +13,9 @@ import (
 	"time"
 
 	"github.com/cenk/backoff"
-	"github.com/giantswarm/certificatetpr"
 	"github.com/giantswarm/microerror"
+	apiv1 "k8s.io/api/core/v1"
```

Comment:
JFYI I saw it usually aliased to `corev1`.

Reply:
I noticed this is inconsistent across the project. So far I simply kept `apiv1` because this is what it was before. We can also boyscout that later in case there is need.